dbus/xdg.c

```diff
@@ -227,7 +228,8 @@ static int handle_notify(sd_bus_message *msg, void *data,
 		return -1;
 	}
 
-	if (expire_timeout < 0 || notif->style.ignore_timeout) {
+	int32_t expire_timeout;
+	if (notif->requested_timeout < 0 || notif->style.ignore_timeout) {
```

Comment:
If this condition isn't satisfied, `expire_timeout` will be uninitialized. We should probably set it to `requested_timeout`.

Reply:
Fixed, good catch. My test notification wasn't specifying one.